Formula/httpd.rb

```diff
@@ -25,7 +25,8 @@ class Httpd < Formula
   def install
     # fixup prefix references in favour of opt_prefix references
     inreplace "Makefile.in",
-      '#@@ServerRoot@@#$(prefix)#', '#@@ServerRoot@@'"##{opt_prefix}#"
+      '#@@ServerRoot@@#$(prefix)#', '#@@ServerRoot@@' \
+                                    "##{opt_prefix}#"
```

Comment:
Not sure this makes sense but ok.

Reply:
It seems like a RuboCop bug, even with the latest version (1.18.2). I'll raise an issue with them.